sklearn/linear_model/tests/test_logistic.py

```diff
@@ -801,26 +802,19 @@ def test_logistic_regression_solvers():
     X, y = make_classification(n_features=10, n_informative=5, random_state=0)
 
     params = dict(fit_intercept=False, random_state=42, multi_class="ovr")
-    ncg = LogisticRegression(solver="newton-cg", **params)
-    lbf = LogisticRegression(solver="lbfgs", **params)
-    lib = LogisticRegression(solver="liblinear", **params)
-    sag = LogisticRegression(solver="sag", **params)
-    saga = LogisticRegression(solver="saga", **params)
-    ncg.fit(X, y)
-    lbf.fit(X, y)
-    sag.fit(X, y)
-    saga.fit(X, y)
-    lib.fit(X, y)
-    assert_array_almost_equal(ncg.coef_, lib.coef_, decimal=3)
-    assert_array_almost_equal(lib.coef_, lbf.coef_, decimal=3)
-    assert_array_almost_equal(ncg.coef_, lbf.coef_, decimal=3)
-    assert_array_almost_equal(sag.coef_, lib.coef_, decimal=3)
-    assert_array_almost_equal(sag.coef_, ncg.coef_, decimal=3)
-    assert_array_almost_equal(sag.coef_, lbf.coef_, decimal=3)
-    assert_array_almost_equal(saga.coef_, sag.coef_, decimal=3)
-    assert_array_almost_equal(saga.coef_, lbf.coef_, decimal=3)
-    assert_array_almost_equal(saga.coef_, ncg.coef_, decimal=3)
-    assert_array_almost_equal(saga.coef_, lib.coef_, decimal=3)
+    solvers = ("newton-cg", "lbfgs", "liblinear", "sag", "saga", "trust-ncg")
+
+    # dict of fitted logistic regressors
```

Comment:
I think we do not think we need the comment here. The code is clear about it's intentions.

We can add a docstring to `test_logistic_regression_solvers` to explain what the test does tho.